### DISCUSSION TITLE
Trouble Emulating Mobile Environment in Cypress: Seeking Help to Locate Visible Elements

### DISCUSSION THREAD

**Discussion by 1vivekbarnwal** [cypress-io/cypress#28271] [Questions and Help]
I've encountered a challenge while attempting to have Cypress emulate a mobile environment, particularly in terms of locating elements that are visible on mobile devices. 
I've experimented with configuring the userAgent and manipulating the viewport, but unfortunately, neither approach has yielded the desired results. I'm seeking advice, insights, or potential solutions from the community. 

Any assistance or guidance in this matter would be highly appreciated.

**Comment by adnanerlansyah403** [comment]
Can you give more details or example for this case ?
